neuromation/api/registry.py

```diff
@@ -14,10 +14,15 @@ class _Registry(_Core):
     """
 
     def __init__(
-        self, session: aiohttp.ClientSession, base_url: URL, token: str, username: str
+        self,
+        session: aiohttp.ClientSession,
+        base_url: URL,
+        token: str,
+        trace_id: Optional[str],
```

Comment:
Why `trace_id` was inserted between `token` and `username`?

Reply:
The constructor is *internal*, the order doesn't matter.
But I've chosen to use session-base_url-token-trace_id sequence because base class `_Core` has this signature